cluster_experiments/power_analysis.py

```diff
@@ -105,26 +262,21 @@ def __init__(
         seed: Optional[int] = None,
         hypothesis: str = "two-sided",
     ):
-        self.perturbator = perturbator
-        self.splitter = splitter
-        self.analysis = analysis
-        self.n_simulations = n_simulations
-        self.target_col = target_col
-        self.treatment = treatment
-        self.control = control
-        self.treatment_col = treatment_col
-        self.alpha = alpha
-        self.hypothesis = hypothesis
-
-        self.cupac_handler = CupacHandler(
+        super().__init__(
+            splitter=splitter,
+            analysis=analysis,
             cupac_model=cupac_model,
             target_col=target_col,
+            treatment_col=treatment_col,
+            treatment=treatment,
+            control=control,
+            alpha=alpha,
             features_cupac_model=features_cupac_model,
+            seed=seed,
+            hypothesis=hypothesis,
         )
-        if seed is not None:
-            random.seed(seed)  # seed for splitter
-            np.random.seed(seed)  # seed for the binary perturbator
-            # may need to seed other stochasticity sources if added
+        self.perturbator = perturbator
+        self.n_simulations = n_simulations
```

Comment:
I think n_simulations should be passed to the AA too, so I would move it to the parent

Reply:
ok! was not aware of this :) In that case if n>1, the power for a given effect should be aggregated like mean(), reflecting the randomness in the split?

Reply:
the idea is averaging standard error, not the power, thuogh I'm not sure if they are equivalent